tests/drivers/motor_driver/Makefile

```diff
@@ -6,7 +6,8 @@ include ../Makefile.drivers_common
 
 USEMODULE += motor_driver
 USEMODULE += shell_cmds_default
-USEMODULE += xtimer
+USEMODULE += ztimer
+USEMODULE += ztimer_sec
```

Comment:
msec might be a better choice  - every timer might be off by one count  for second -> 3-4 seconds  for msec 3000 - 3001 milliseconds

( usually it is a good choice to not wait single digits if you care about the time you wait)

Reply:
done